plaso/cli/helpers/elastic_output.py

```diff
@@ -67,6 +69,11 @@ def AddArguments(cls, argument_group):
         '--elastic_user', dest='elastic_user', action='store',
         default=cls._DEFAULT_ELASTIC_USER, help=(
             'Username to use for Elasticsearch authentication.'))
+    argument_group.add_argument(
+        '--elastic_password', dest='elastic_password', action='store',
+        default=cls._DEFAULT_ELASTIC_PASSWORD,
+        help=('Password to use for Elasticsearch authentication. Can '+\
```

Comment:
Nit: trailing `+\` is not needed bounding parenthesis `(...)` will make it 1 string.